homeassistant/components/wolflink/__init__.py

```diff
@@ -0,0 +1,93 @@
+"""The Wolf SmartSet Service integration."""
+from _datetime import timedelta
+import logging
+
+from httpcore._exceptions import ConnectError
```

Comment:
The import is supposed to be done directly from the `httpcore` package, not from the `_exceptions` submodule.

https://github.com/encode/httpcore/blob/44d441375234272d06b8e5bc33310398bb6746d8/httpcore/__init__.py#L37

But since the wolflink library is using the httpx library, we should instead import from that library.

https://github.com/encode/httpx/blob/3f96e74961f9faf013e5aff6c73a71f3b7bc2676/httpx/__init__.py#L64

Reply:
Thanks for that. Python is not my "native" language so I am still learning and thank to you even more :)

Reply:
It is not yet released in newer version of httpx. So I will add import from httpcore.